### PR TITLE
[13] 2022/related-links-update

### PR DESCRIPTION
**Pull request by zastrow**

### CODE REVIEW

File: src/components/2022/related-article.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const RelatedArticle = ({ image, title, description, href }) => {
+  return (
+    <a className="cmp-related-reading__link" href={href}>
+      <Grid gap="lg" gapColMD="1xl">
+        <GridCell spanSM={2} spanMD={2}>
+          <img src={image} alt="" className="cmp-related-reading__image" />
+        </GridCell>
+        <GridCell spanSM={4} startSM={3} spanMD={5} startMD={3} spanLG={6}>
+          <h4 className="cmp-type-body-xlarge util-margin-all-none util-weight-regular">{title}<svg className="cmp-related-reading__arrow" width="14" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.703 6.36H1.058v9.16" stroke="currentColor" stroke-width="1.5"/><path d="m7.383 1.52 4.675 4.838-4.675 4.84" stroke="currentColor" stroke-width="1.5"/></svg></h4>
```
Comment (catheraaine) [inline src/components/2022/related-article.js:14]: stick an aria-hidden on your svg

File: src/scss/2022/components/_related-reading.scss [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
+    }
   }
   
   &__image {
-    box-shadow: 0 0 0 0.125rem var(--color-black),0 0 0 0 var(--color-white);
+    box-shadow: 0 0 0 0 var(--color-black), 0 0 0 0 var(--color-dark);
     transition: box-shadow var(--default-duration) var(--default-timing);
   }
   
   &__link:hover &__image {
```
Comment (catheraaine) [inline src/scss/2022/components/_related-reading.scss:23]: nice

File: src/sections/2022/section-4.mdx [hunk 3]
```diff
@@ -45,19 +46,18 @@ import ComarisonTable from "../../components/2022/comparison-table"
...
-    image="https://sparkbox.com/uploads/featured_images/case-study_t-barney_22-03.png"
-    href="https://sparkbox.com/foundry/design_system_action_plan_case_study_building_a_design_system_foundation"
-  />
-  
+  <RelatedReading>
+    <RelatedArticle
+      title="Why Your Design System Needs a Documentation Site"
+      description="Adoption has been a top challenge for many design systems over the past five years. If your team is struggling with adoption, one way to address it is to make your documentation more user-friendly."
+      image="/images/2022/accelerate-adoption.svg"
+      href="https://sparkbox.com/foundry/how_a_documentation_site_can_increase_your_design_system_subscribers" />
```
Comment (catheraaine) [inline src/sections/2022/section-4.mdx:54]: ha, it me!

**Review by catheraaine — CHANGES_REQUESTED** [review]
Overall looks and feels good! I think I have one change request, maybe two

Screen Shot 2022-06-14 at 10 56 59 AM

File: src/components/2022/related-article.js (hunk 1 above)
Comment (zastrow) [reply src/components/2022/related-article.js:14]: addressed

Comment (lunarfusion) [reply src/components/2022/related-article.js:14]: I like the foundry images in white. Very Louise Nevelson.
Screen Shot 2022-06-14 at 11 32 37 AM

Comment (catheraaine) [reply src/components/2022/related-article.js:14]: ohhhh for real!

**Review by catheraaine — APPROVED** [review]
Perfect!

**Pull request closed without merging**